src/Redirection/DynamicRedirectionsSubscriber.php

```diff
@@ -3,22 +3,36 @@
 namespace AppBundle\Redirection;
 
 use AppBundle\Entity\Redirection;
+use AppBundle\Repository\EventRepository;
 use AppBundle\Repository\RedirectionRepository;
 use Symfony\Component\EventDispatcher\EventSubscriberInterface;
 use Symfony\Component\HttpFoundation\RedirectResponse;
+use Symfony\Component\HttpFoundation\Response;
 use Symfony\Component\HttpKernel\Event\GetResponseForExceptionEvent;
 use Symfony\Component\HttpKernel\Exception\NotFoundHttpException;
+use Symfony\Component\Routing\Matcher\UrlMatcherInterface;
 
 /**
  * Handle dynamic redirections editable in the administration panel.
  */
 class DynamicRedirectionsSubscriber implements EventSubscriberInterface
 {
-    private $repository;
+    const REDIRECTIONS = [
+        '/evenements/' => '/evenements',
+        '/comites/' => '/comites',
+    ];
 
-    public function __construct(RedirectionRepository $repository)
+    private $redirectRepository;
+
+    private $router;
+
+    private $eventRepository;
+
+    public function __construct(RedirectionRepository $redirectionRepository, EventRepository $eventRepository, UrlMatcherInterface $router)
```

Comment:
also I would suggest changing the name to `$urlMatcher` instead of `$router` everywhere.